includes/settings/js/src/components/CreateContentModel.jsx

```diff
@@ -5,7 +5,7 @@ import { useHistory } from "react-router-dom";
 import { ModelsContext } from "../ModelsContext";
 import { insertSidebarMenuItem } from "../utils";
 import { useInputGenerator } from "../hooks";
-import { toSanitizedKey } from "../formats";
+import { toSanitizedKey, toValidApiId } from "../formats";
```

Comment:
It looks like `toSanitizedKey` is an unused import now, and could be removed here.

```suggestion
import { toValidApiId } from "../formats";
```